extern/iso_c_fortran_bmi/test/test_iso_c.c

```diff
@@ -204,6 +213,45 @@ int main(int argc, char** argv)
     printf("get_value_double INPUT_VAR_1: %f\n", value_d);
     check_status(&status, "get_value_double");
 
+    value = -2;
+    int *value_ptr = &value;
+    status = get_value_ptr_int(&bmi_handle, "INPUT_VAR_3", value_ptr);
+    printf("get_value_ptr_int: %d\n", value_ptr);
+    //printf("get_value_ptr_int: %d\n", *value_ptr);  //Can uncomment this line if "dest_ptr != c_null_ptr" in iso_c_bmi.f90
+    check_status(&status, "get_value_ptr_int");
```

Comment:
Perhaps a new function is needed here to check for `BMI_FAILURE` as a "correct" behaviour.

```C
void check_failure(int* status, char* name) {
    if( *status == BMI_FAILURE ){
        printf("EXPECTED FAILURE\n");
    }
    else {
        printf("FAILURE TO FAIL\n");
        exit(-1);
    }
}
```
then call this when you are expecting a failure.